packages/web/src/scss/components/Modal/index.html

```diff
@@ -34,7 +34,9 @@
             aria-controls="modal-example-1"
             aria-expanded="false"
           >
-            <span class="Modal__close" aria-hidden="true"></span>
+            <svg class="Icon" width="24" height="24">
```

Comment:
What about that `aria-hidden`?

Reply:
Added. Also in web-twig. So check it, please.